### PR TITLE
Feat: Create the type file for @nkzq/copy-paste.

### PR DESCRIPTION
**Pull request by sfilata**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## nkzq__copy-paste ([<kbd>unpkg</kbd>](https://unpkg.com/browse/nkzq__copy-paste@latest/))
was missing the following properties:
1. silent
1. noConflict
1. global
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 8106a74336b24d8efacee6dfab45d9fb339a6ef5
</p>

File: types/nkzq__copy-paste/nkzq__copy-paste-tests.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import * as CopyPaste from 'copy-paste';
```
Comment (peterblazejewicz) [inline types/nkzq__copy-paste/nkzq__copy-paste-tests.ts:1]:
```suggestion
import * as CopyPaste from '@nkzq/copy-paste';
```
you meant this? 
Also you probably want `paths` property to be set, as this is namespaced package:
https://github.com/DefinitelyTyped/DefinitelyTyped#what-about-scoped-packages

File: types/nkzq__copy-paste/tslint.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "@definitelytyped/dtslint/dt.json",
+  "rules": {
+      "no-consecutive-blank-lines": false,
+      "no-redundant-jsdoc-2": false,
+      "no-trailing-whitespace": false,
+      "strict-export-declare-modifiers": false,
+      "unified-signatures": false
```
[the remark this answers is not in the document]
Comment (sfilata) [reply types/nkzq__copy-paste/tslint.json]: Emmm, I copied the origin 'copy-paste' d.ts from the official code. I've already removed it. Please check it again. Thanks.

**Review by jakebailey — APPROVED** [review]

**Comment by sfilata** [comment]
Ready to merge.

**Pull request merged**